src/v/redpanda/application.cc

```diff
@@ -411,7 +411,8 @@ void application::setup_public_metrics() {
       {{"io_queue_total_read_ops", ssx::metrics::public_metrics_handle},
        {"io_queue_total_write_ops", ssx::metrics::public_metrics_handle},
        {"memory_allocated_memory", ssx::metrics::public_metrics_handle},
-       {"memory_free_memory", ssx::metrics::public_metrics_handle}})
+       {"memory_free_memory", ssx::metrics::public_metrics_handle},
+       {"reactor_utilization", ssx::metrics::public_metrics_handle}})
```

Comment:
Hmm.  I think anything with `reactor` in the name is definitely too implementation-detailed for the public metrics.

Internally to seastar it looks like this one is a decaying load statistic, whereas our currently redpanda_cpu_busy_seconds_total metric is a giving a direct measurement of how fast the process is accumulating busy time (i.e. take the gradient and you get how busy the CPU was in your sample period).

So two questions:
- A) do we need this?
- B) how do we explain to the user which statistic is the right one for a dashboard that wants to show "how busy is my CPU" in one chart?

Reply:
I agree that the name is too specific, but the metrics API doesn't allow renaming at the moment (maybe it should; our seastar metrics implementation has diverged quite a bit from upstream).

Done a bit of research and `reactor_utilization` seems to be the most accurate metric of how much work is going on at any given point. For some reason I didn't think of using the rate of change in `redpanda_cpu_busy_seconds_total` to estimate this. Let me chat to Tristan and see if that would work.

Reply:
Endede up removing it. We'll got with the rate of `redpanda_cpu_busy_seconds_total`.